styles/rentry/catppuccin.user.css

```diff
@@ -0,0 +1,526 @@
+      );
+      content: url("data:image/svg+xml,@{svg}");
+    }
+    // code
+    .highlighttable {
+      border-left: 4px solid @surface0 !important;
+    }
+    .highlight {
+      border-color: @surface0 !important;
+    }
+    .linenodiv > pre > .normal > a:not(:hover),
+    .linenodiv > pre > a:not(:hover) {
+      color: @overlay2 !important;
+    }
+    .entry-text pre,
+    .markdownx-preview pre,
+    code {
+      color: @red !important;
+    }
+    .highlight .n,
+    .highlight .nx {
+      color: @text;
+    }
+    .highlight .hll {
+      background-color: @yellow;
+    }
+    .highlight .p {
+      color: @peach;
+    }
+    .highlight .cp {
+      color: @peach;
+    }
+    .highlight .gd {
+      color: @red;
+    }
+    .highlight .gr {
+      color: @red;
+    }
+    .highlight .gi {
+      color: @green;
+    }
+    .highlight .go {
+      color: @overlay0;
+    }
+    .highlight .gu {
+      color: @mauve;
+    }
+    .highlight .gt {
+      color: @sapphire;
+    }
+    .highlight .kt {
+      color: @pink;
+    }
+    .highlight .na {
+      color: @yellow;
+    }
+    .highlight .no {
+      color: @peach;
+    }
+    .highlight .ni {
+      color: @subtext0;
+    }
+    .highlight .ne {
+      color: @maroon;
+    }
+    .highlight .nl {
+      color: @yellow;
+    }
+    .highlight .nn {
+      color: @mauve;
+    }
+    .highlight .w {
+      color: @subtext1;
+    }
+    .highlight .sd {
+      color: @blue;
+    }
+    .highlight .se {
+      color: @peach;
+    }
+    .highlight .c,
+    .highlight .ch,
+    .highlight .cm,
+    .highlight .cpf,
+    .highlight .c1,
+    .highlight .cs {
+      color: @overlay2;
+    }
+    .highlight .k,
+    .highlight .kc,
+    .highlight .kd,
+    .highlight .kn,
+    .highlight .kp,
+    .highlight .kr,
+    .highlight .nb,
+    .highlight .nt,
+    .highlight .sx,
+    .highlight .bp {
+      color: @maroon;
+    }
+    .highlight .o,
+    .highlight .m,
+    .highlight .mb,
+    .highlight .mf,
+    .highlight .mh,
+    .highlight .mi,
+    .highlight .mo,
+    .highlight .il {
+      color: @teal;
+    }
+    .highlight .gh,
+    .highlight .gp {
+      color: @sapphire;
+    }
+    .highlight .s,
+    .highlight .sa,
+    .highlight .sb,
+    .highlight .sc,
+    .highlight .dl,
+    .highlight .s2,
+    .highlight .sh,
+    .highlight .s1 {
+      color: @green;
+    }
+    .highlight .nc,
+    .highlight .nf,
+    .highlight .fm {
+      color: @sky;
+    }
+    .highlight .nd,
+    .highlight .ow {
+      color: @mauve;
+    }
+    .highlight .nv,
+    .highlight .ss,
+    .highlight .vc,
+    .highlight .vg,
+    .highlight .vi,
+    .highlight .vm {
+      color: @lavender;
+    }
+    .highlight .si,
+    .highlight .sr {
+      color: @red;
+    }
```

Comment:
Nesting please.

Reply:
Also please make sure you haven't just taken their colors and made them Catppuccin- syntax highlighting needs to match how our style guide is, so look at how code looks in (paste it in to) VS Code or Neovim or whatever your editor is and try to match the classes as well as you can.

Reply:
i tried to match the colors in [`1e1ec94` (#1198)](https://github.com/catppuccin/userstyles/pull/1198/commits/1e1ec9444e82f53af5578d20bf1fd4fb1e3fd99f) with the vscode(/ium) theme and it became more accurate however there are things i can't solve such as special characters (`(){}.<>=*:;`) being in the same color due to using the same class (`.p`) ([style-guide.md](https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md#code-editors) says that symbols and atoms(?) should be red so i made it red)

if you notices any issues with the syntax highlighting please point them out